### PR TITLE
Extend timeout for first test in chunk

### PR DESCRIPTION
**Pull request by zsperske**
If the apk for a test chunk is not installed on the device yet, extend the timeout to cover install with retries.

### CODE REVIEW

File: torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt [hunk 1]
```diff
@@ -92,6 +94,17 @@ class TestRunFactory {
         }
     }
 
+    private fun getTimeoutMillis(args: Args, installer: Installer, testChunk: TestChunk): Long {
+        val chunkTimeOutWithRetries = TimeUnit.SECONDS.toMillis(args.chunkTimeoutSeconds) * args.retriesPerChunk
+        val installTimeOutWithRetries = TimeUnit.SECONDS.toMillis(args.installTimeoutSeconds.toLong()) * args.retriesInstallPerApk
```
Comment (ScottBishop) [inline torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt]: Should this be moved into the else? Only needs to be calculated in that case

Comment (zsperske) [reply torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt]: It's a trivial calculation and I think it reads better with them grouped up here.

File: torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt [hunk 2]
```diff
@@ -33,7 +33,7 @@ class TestRunFactory {
             testChunkRunner: TestChunkRunner = TestChunkRunner(adbDevice, logcatFileIO, installer)
     ): Single<AdbDeviceTestSession> {
         val testSession = AdbDeviceTestSession(adbDevice = adbDevice,
-                                               logcatFile = logcatFileIO.fullLogcatFile)
+                logcatFile = logcatFileIO.fullLogcatFile)
```
Comment (hujim) [inline torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt]: revert?

File: torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt [hunk 3]
```diff
@@ -60,9 +60,9 @@ class TestRunFactory {
                     }
 
                     adbDevice.log("Device Test Session complete, " +
-                                "${testSession.passedCount} passed, " +
-                                "${testSession.failedCount} failed, took " +
-                                "${testSession.durationMillis.millisToHoursMinutesSeconds()}."
+                            "${testSession.passedCount} passed, " +
+                            "${testSession.failedCount} failed, took " +
+                            "${testSession.durationMillis.millisToHoursMinutesSeconds()}."
```
Comment (hujim) [inline torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt]: revert?

File: torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt [hunk 4]
```diff
@@ -142,9 +155,9 @@ class TestRunFactory {
                     .map { TestDetails(it.className, it.testName) }
                     .forEach { testDetails: TestDetails ->
                         filePuller.pullTestFolder(args.testFilesPullDeviceDirectory,
-                                                  args.testFilesPullHostDirectory,
-                                                  testDetails,
-                                                  pullFileTimeout).await()
```
Comment (hujim) [inline torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt]: revert?

File: torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt (hunk 1 above)
Comment (hujim) [reply torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt]: Both works, but if-else brackets :P?

Comment (hujim) [reply torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt]: fun result, install retries happen within each chunk retry.
So it's actually 
```
val installTimeOutWithRetries = TimeUnit.SECONDS.toMillis(args.installTimeoutSeconds.toLong()) * args.retriesPerChunk * args.retriesInstallPerApk
```

**Pull request merged**